topics/babel/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Sebastian McKenzie, James Kyle, Henry Zhu, Logan Smyth, Daniel Tschinder
+display_name: Babel
+github_url: https://github.com/babel
+logo: babel.png
+released: Oct 1, 2014
```

Comment:
Birthday/first commit was Sep 28, 2014, I would say that could count as the first release too?

Reply:
I based the "released" value on the publish date of 6to5 on npm. Happy to change it though. 🤷‍♂️

Reply:
Let's go with first commit, since we celebrate that 🎂

Reply:
Done :+1: